deploy/internal/statefulset-core.yaml

```diff
@@ -33,6 +33,13 @@ spec:
           secret:
             secretName: noobaa-s3-serving-cert
             optional: true
+        - name: oidc-token
```

Comment:
do we always want to mount this?

Reply:
I think we tested that for the case where it does not exist and nothing harmful happened.
@ezio-auditore Can you please confirm?